src/vscode-dts/vscode.proposed.interactive.d.ts

```diff
@@ -192,6 +197,6 @@ declare module 'vscode' {
 
 		export function registerInteractiveEditorSessionProvider(provider: InteractiveEditorSessionProvider): Disposable;
 
-		export function transferChatSession(session: InteractiveSession, toWorkspace: Uri): void;
+		export function transferChatSession(session: InteractiveSession, transferredSessionState: InteractiveTransferredSessionState): void;
```

Comment:
TBH I would not expect the input value to be something that the extension has to include, shouldn't it just be automatically done by core?

Reply:
I thought about saving the input value with the session id and restoring from there from core. What if the input value needs to be something else other than what was previously used going from /create* to /modify* which the provider wants to setup?

Reply:
Why would we need the input to be something else though?